pkgs/by-name/cl/cloudquery/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  stdenvNoCC,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "cloudquery";
+  version = "6.11.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudquery";
+    repo = "cloudquery";
+    rev = "refs/tags/cli-v${version}";
+    hash = "sha256-zVzHPAN/UAVQF56qcoIFOqtX/Aek3+1lO2O965UYxPM=";
+  };
+
+  vendorHash = "sha256-9YvsVflHr4QSVCYCcWg1Okmncc1sNteMJEAdz92BIgI=";
+
+  CGO_ENABLED = 0;
+
+  modRoot = "cli";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/cloudquery/cloudquery/cli/cmd.Version=${version}"
+  ];
+
+  postInstall =
+    ''
+      mv $out/bin/cli $out/bin/cloudquery
+      rm $out/bin/gen
+    ''
+    + lib.optionalString (stdenvNoCC.buildPlatform.canExecute stdenvNoCC.hostPlatform) ''
+      installShellCompletion --cmd cloudquery \
+        --bash <($out/bin/cloudquery completion bash) \
+        --fish <($out/bin/cloudquery completion fish) \
+        --zsh <($out/bin/cloudquery completion zsh)
+    '';
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
```

Comment:
After updating to 6.12.9 `--version` will print `cloudquery version development`, making the build fail with `versionCheckHook` because it can't find the the version number. I guess by adding some `ldflags` this can be fixed.